### PR TITLE
Nextjs Aceternity UI sample

### PR DESCRIPTION
**Pull request by AkashJana18**
GlobalSync frontend

### CODE REVIEW

File: GlobalSync-frontend/README.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+**GlobalSync** is an innovative platform designed to empower remote teams to work seamlessly across different time zones. By providing intuitive tools for time zone coordination, collaboration, and scheduling, GlobalSync ensures productivity and convenience for global teams.
+
+# 🚀 Getting Started  
+
+### Open Using Daytona  
+
+1. **Install Daytona**: Follow the [Daytona installation guide](https://www.daytona.io/docs/installation/installation/).  
+2. **Create the Workspace**:  
+   ```bash  
+   daytona create https://github.com/AkashJana18/sample-nextjs-aceternity.git
```
Comment (Tpuljak) [inline GlobalSync-frontend/README.md]: The URL is wrong here. It needs to be of this repo (https://github.com/daytonaio/sample-nextjs-aceternity)

**Review by Tpuljak — CHANGES_REQUESTED** [review]
Please also sign your commit

**Comment by Tpuljak** [comment]
@AkashJana18 you need to sign at least one commit

**Comment by AkashJana18** [comment]
> @AkashJana18 you need to sign at least one commit

Yes, I was doing that. You replied pretty fast even before i dropped any message.
Really appreciate your quick response! ✨

**Pull request merged**